go.mod

```diff
@@ -0,0 +1,9 @@
+module github.com/segmentio/analytics-go
+
+go 1.14
+
+require (
+	github.com/segmentio/backo-go v0.0.0-20200129164019-23eae7c10bd3
+	github.com/segmentio/conf v1.2.0
+	github.com/xtgo/uuid v0.0.0-20140804021211-a0b114877d4c
```

Comment:
Maybe a change for a subsequent PR, we could replace this dependency with `github.com/google/uuid` which is a preferred UUID package.

Reply:
Noted, and can do some time this week.